pkg/pit/dnsmasq.go

```diff
@@ -70,7 +70,7 @@ server=/hmn/
 address=/hmn/
 domain=hmn,{{.CIDR.IP}},{{.DHCPEnd}},local
 interface-name=pit.hmn,bond0.hmn0
-dhcp-option=interace:bond0.hmn0,option:domain-search,hmn
+dhcp-option=interface:bond0.hmn0,option:domain-search,hmn
```

Comment:
> Does this actually impact us?

It looks like the error is just to the domain-search option on the HMN interfaces. And I incorrectly described this as a kernel parameter like -- this is the dnsmasq configuration, I believe. So the impact I think if that dnsmasq is going to give incorrect domain-search info to requests on the HMN network. But that's my take on this as someone who is fairly novice when it comes to dnsmasq and the inner workings of CSI, so...

Reply:
The line is specifying two separate and unrelated options.  `interface:bond0` tells dhcp to only listen on that interface.  Since it's misspelled, it will default to listening on all interfaces.

Reply:
> The line is specifying two separate and unrelated options. `interface:bond0` tells dhcp to only listen on that interface. Since it's misspelled, it will default to listening on all interfaces.

So it won't cause anything to fail, sounds like, but I'm guessing customers don't assume it'll be listening on all interfaces. I guess for our very security-minded customers this could potentially be alarming, but who can say?